tools/nanopolish/nanopolish_methylation.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="nanopolish_methylation" name="Nanopolish methylation" version="0.1.0">
+    <description>- Classify nucleotides as methylated or not. </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input_merged' reads.fasta && 
+        
+        #if $input_reads_raw.extension == 'fast5':
+            mkdir fast5_files && ln -s '$input_reads_raw' fast5_files/read1.fast5 &&
+        #else
+            ln -s '$input_reads_raw' fast5_files.tar.gz &&
+            mkdir fast5_files && tar -xzf fast5_files.tar.gz -C fast5_files &&
+        #end if
+
+        nanopolish index -d fast5_files/ reads.fasta &&
+        ln -s '$input_reads_bam' reads.bam &&
+        ln -s '${input_reads_bam.metadata.bam_index}' reads.bam.bai &&
+        ln -s '$input_genome_fa' genome.fa &&
+        
+        nanopolish call-methylation
+        -r reads.fasta
+        -b reads.bam
+        -g genome.fa
+        $verbose
+        #if $window and str($window).strip():
+          -w "${window}"
+        #end if  
+        > methylation_calls.tsv
+    ]]></command>
+    <inputs>
+      <!-- index inputs -->
+        <param type="data" name="input_merged" format="fasta,fastq" label="Basecalled merged reads.fa"/>
+        <param type="data" name="input_reads_raw" format="h5,fast5.tar.gz" label="Flat archive file of raw fast5 files"/>
+
+        <!-- variants consensus inputs -->
+        <param type="data" name="input_reads_bam" format="bam" label="Reads aligned to the reference genome" help="(-b)" />
+        <param type="data" name="input_genome_fa" format="fasta" label="The reference genome" help="(-g)"/>
+
+        <param name="window" type="text" optional="true" label="find variants in window chromsome_name:start-end" help="(-w)"/>
```

Comment:
Is this more a region than a window ... in the explanation? also the `_` is strange

Reply:
Maybe we can write at least in the help section, that this is a simple "region"?